char status
int16 id